### PR TITLE
GUI: Re-order config settings

### PR DESCRIPTION
**Pull request by RedDevilus**
### Description of Changes

Re-order config setting to older order.

Master:
image

PR:
image

### Rationale behind Changes

Visual clarity between option and also muscle memory.

### CODE REVIEW

**Comment by arcum42** [comment]
Seriously, the whole thing is a mess anyways. Eight separate setting dialog boxes in eight different menu items? And how many times can we use the word "settings" in a menu?

We really need to combine a lot of these...

**Comment by RedDevilus** [comment]
Yeah, a poll would help in this case.

**Comment by arcum42** [comment]
If we were just going to do some general renaming, I'd suggest something like:
General...
Memory cards...
Bios...
--
Graphics...
Sound...
Gamepads...
Network/Hard drive...
Usb...
--
Multitap 1
Multitap 2
--
Language...
Reset to defaults

Though I'm sure all the translations would break. Language *could* just be a menu, too...

**Comment by RedDevilus** [comment]
Languages are held back anyway, so by anything they are already broken. So don't hold back if you have suggestions.

**Comment by Shoegzer** [comment]
@arcum42 Out of curiosity, are all the ellipses necessary?  Just seems cleaner without them.

**Comment by arcum42** [comment]
> @arcum42 Out of curiosity, are all the ellipses necessary? Just seems cleaner without them.

It's a standard ui convention that if a menu item opens a dialog box, it has an ellipse after it, and all of those open dialog boxes.

**Comment by tellowkrinkle** [comment]
While you're at it, maybe move the `&` on `GamePad Settings` to `Game&Pad Settings` to reduce conflict?

> Though I'm sure all the translations would break.

There aren't any to break
screenshot broken translations

**Comment by RedDevilus** [comment]
image

Macro is fine now as well.

**Comment by RedDevilus** [comment]
image
Removed the punctations marks/ dots.

**Comment by RedDevilus** [comment]
image
Added settings back and removed the dots for about.

**Comment by RedDevilus** [comment]
4446 Current Design
Hopefully this is fine unless someone has some more input. It should be ready for merging.

File: pcsx2/gui/AppRes.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@
 RecentIsoList::RecentIsoList(int firstIdForMenuItems_or_wxID_ANY)
 {
 	Menu = std::unique_ptr<wxMenu>(new wxMenu());
-	Menu->Append(MenuId_IsoBrowse, _("Browse..."), _("Browse for an ISO that is not in your recent history."));
+	Menu->Append(MenuId_IsoBrowse, _("Browse"), _("Browse for an ISO that is not in your recent history."));
```
Comment (lightningterror) [inline pcsx2/gui/AppRes.cpp:45]: The dots should stay here.

File: pcsx2/gui/MainFrame.cpp [hunk 2]
```diff
@@ -61,7 +61,7 @@ wxMenu* MainEmuFrame::MakeStatesSubMenu(int baseid, int loadBackupId) const
 	// baseid in the negatives will order in a different section, so if you want to increase more slots you can still easily do this, as -1 it will have the same function as opening file for savestates, which is bad
 	// For safety i also made them inactive aka grayed out to signify that's it's only for informational purposes
 	// Fixme: In the future this can still be expanded to actually cycle savestates in the GUI.
-	mnuSubstates->Append(baseid - 1, _("File..."));
+	mnuSubstates->Append(baseid - 1, _("File"));
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:64]: Here too.

File: pcsx2/gui/MainFrame.cpp [hunk 3]
```diff
@@ -374,7 +374,7 @@ void MainEmuFrame::CreatePcsx2Menu()
 		_("Wipes all internal VM states."));
 	m_menuSys.FindItem(MenuId_Sys_Shutdown)->Enable(false);
 
-	m_menuSys.Append(MenuId_Boot_ELF, _("&Run ELF..."),
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:377]: Ditto.

File: pcsx2/gui/MainFrame.cpp [hunk 4]
```diff
@@ -506,7 +507,7 @@ void MainEmuFrame::CreateCaptureMenu()
 	keyCode.Shift(false);
 	keyCode.Cmd(false);
 	AppendShortcutToMenuOption(*sysScreenShotItem, keyCode.toTitleizedString());
-	m_submenuScreenshot.Append(MenuId_Capture_Screenshot_Screenshot_As, _("Screenshot As..."));
+	m_submenuScreenshot.Append(MenuId_Capture_Screenshot_Screenshot_As, _("Take Screenshot And Browse"));
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:510]: Save Screenshot As... ?

**Pull request merged**